### ISSUE TITLE
Color picker text boxes’ border has insufficient contrast against is adjacent color in both focused and not focused state

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#874]
The following AI-windows mode(s) have this issue:
- Regular mode
- High Contrast White mode
- High Contrast Black mode
- High Contrast ```#2``` mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![21 1](https://user-images.githubusercontent.com/22068434/88356888-3b8c8800-cd1e-11ea-9c02-c0ca0644b6ca.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this has been fixed (with the exception of the focused state in regular mode) in the release at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1391.01. Please validate and close at your convenience. For the focus state in regular mode, I remember that we had a similar issue with the edit edit control in the settings page. These should be lumped together, but I'm not sure if there's already an issue tracking that or not. Thanks!

**Comment by iamrafan** [comment]
@DaveTryon, I haven't yet filed the issue for focused state. I shall reference it here once I file it. Rest of the states/modes look good, Validated them  in Canary Release: v1.1.1398.01.
